applications/dashboard/controllers/api/UsersApiController.php

```diff
@@ -598,6 +598,35 @@ public function post_requestPassword(array $body) {
         $this->validateModel($this->userModel, true);
     }
 
+    /**
+     * @param $id
+     * @param array $body
+     * @return array $result The response body
+     * @throws Exception
+     * @throws NotFoundException if unable to find the user.
+     */
+    public function post_confirmEmail ($id, array $body) {
+        $this->permission('Garden.Users.Edit');
+
+        $in = $this->schema( ['emailKey:s' => 'key generated on registration'])->setDescription('Confirm new user email upon registration');
+        $out = $this->schema(['emailConfirmed:b' => 'The current email confirmed value.'], 'out');
```

Comment:
`EntryController::confirmEmail` includes user ID and the actual email address that was confirmed, in addition to emailConfirmed. Let's include those in the output too.